depcheck.config.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
     },
     project: ['tsconfig.base.json'],
   },
-  plugins: ['@typescript-eslint'],
+  plugins: ['@typescript-eslint', 'spellcheck', 'import', 'jsdoc', 'prefer-arrow'],
```

Comment:
This change ensures we can run the package.json check when there are excludes for things like `no-cycle`.